js/ui/screenShield.js

```diff
@@ -164,7 +164,7 @@ var ScreenShield = class {
 
             // The time may have been extended using the tray applet from a
             // logged in session.
-            if (this._dialog === null)
+            if (!this._dialog || this._dialog === null)
```

Comment:
I think you can just remove the null check in this case.

Reply:
Yeah you're right, just checked in gjs
```
↳ gjs
gjs> this._dialog
typein:1:1 strict warning: reference to undefined property "_dialog"
gjs> if (!this._dialog) log("hello");
Gjs-Message: 17:49:31.504: JS LOG: hello
gjs> this._dialog = null;
null
gjs> if (!this._dialog) log("hello");
Gjs-Message: 17:49:39.775: JS LOG: hello
gjs> if (this._dialog === null) log("hello");
Gjs-Message: 17:50:07.525: JS LOG: hello
gjs> 
```